prow/plugins/label/label.go

```diff
@@ -194,9 +197,19 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 		}
 		if err := gc.AddLabel(ae.org, ae.repo, ae.number, sigLabel); err != nil {
 			log.WithError(err).Errorf("Github failed to add the following label: %s", sigLabel)
+		} else if !isMember {
+			toRepeat = append(toRepeat, sigMatch[0])
 		}
 	}
 
+	if len(toRepeat) > 0 {
+		msg := fmt.Sprintf(chatBack, strings.Join(toRepeat, ", "))
```

Comment:
Correct me if i am wrong.
This message will run like: 
`Reiterating the mentions to trigger a notification: \n @kubernetes/sig-bugs, @kubernetes/sig-misc`